UI-III-Flexbox/index-UI3.css

```diff
@@ -0,0 +1,273 @@
+}
+
+header nav a {
+	/*border: 1px solid blue;*/
+	margin: 0 auto;
+}
+
+header .logo {
+	/*border: 1px solid blue;*/
+	position: relative;
+    bottom: 10px;
+}
+
+.services-header-img {
+	/*border: 1px solid red;*/
+	position: relative;
+	display: flex;
+    justify-content: center;
+    align-items: center;
+	margin: 0 auto;
+margin-bottom: 30px;
+	width: 90%;
+}
+
+.top-content {
+    /*border: 5px solid red;*/
+    width: 100%;
+}
+.top-content .text {
+	/*border: 1px solid red;*/
+    width: 80%;
+    padding-left: 70px;
+}
+
+
+hr {
+	width: 100%;
+    border-width: 1px;
+    border-color: black;
+	margin: 30px 45px 0px 45px;
+}
+
+.middle-content {
+	border-bottom: 5px solid black;
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+	width: 880px;
+    margin: 0 35px;
+    /*justify-content: space-evenly;*/
+    flex-direction: row;
+}
+
+
+.middle-content .box-1 {
+	border: 1px solid rgb(0, 0, 0);
+    width: 360px;
+    margin: 30px 50px 0 10px;
+	padding: 20px;
+    background-color: whitesmoke;
+}
+
+.middle-content .box-2 {
+	border: 1px solid rgb(0, 0, 0);
+	width: 360px;
+	margin: 30px 0 0 10px;
+	padding: 20px;
+    background-color: whitesmoke;
+}
+
+.middle-content .box-3 {
+	border: 1px solid rgb(0, 0, 0);
+    width: 360px;
+    margin: 30px 50px 0 10px;
+	padding: 20px;
+    background-color: whitesmoke;
+}
+
+
+.middle-content .box-4 {
+	border: 1px solid rgb(0, 0, 0);
+    width: 360px;
+	margin: 30px 0 0 10px;
+	padding: 20px;
+    background-color: whitesmoke;
+}
+
+.middle-content .box-5 {
+	border: 1px solid black;
+    width: 360px;
+    margin: 30px 50px 30px 10px;
+	padding: 20px;
+    background-color: whitesmoke;
+}
+
+.middle-content .box-6 {
+	border: 1px solid black;
+	width: 360px;
+	margin: 30px 0 30px 10px;
+	padding: 20px;
+    background-color: whitesmoke;
+}
+
+
+button {   
+	color: black;
+    width: 130px;
+    height: 35px;
+    padding-left: 10px;
+	border-radius: 3px;
+	margin-top: 40px;
+    background-color: white;
+    border: 1px solid black;
+}
+
+
+.services-info-img {
+	position: relative;
+    border: 1px solid rgb(0, 0, 0);
+}
+
+
+.facts {
+	display: flex;
+	flex-direction: row;
+    justify-content: space-evenly;
+    margin: 20px 0px 0px 65px;
+	border-top: 3px solid black;
+	border: 1px solid rgb(0, 0, 0);
+	padding: 30px;
+	width: 90%;
+    margin: 20px 0px 0px 70px;
+}
+
+.service-facts {
+    margin-top: -120px;
+    text-align: center;
+    float: left;
+    width: 50%;
+    height: 140px;
+    display: inline-block;
+    text-align: left;
+    padding-top: 100px;
+
+}
+
+.right-side {
+    margin-top: 200px;
+    margin: 0 100px 0 500px;
+    /*flex: end;*/
+}
+
+.services-info-img {
+    margin: 0 0 0 -40px;
+    width: 400px;
+}
+
+.row {
+    margin: 30px 0 0 0;
+}
+
+
+footer {
+    margin: 0 45px;
+    
+}
+
+.facts h2 {
+	margin-top: 20px;
+}
+
+ul {
+    list-style-type: square;
+    line-height: 20pt;
+}
+
+.copy {
+    margin-top: 40px;
+    text-align: center;
+}
```

Comment:
its good practice to have all css in just the index file. you can add comments to separate the sections/pages. this way you don't have to continuously rewrite the header and footer styling and once you change it 1 time it changes on all pages so there's more consistency